kickoff/fennecversions.py

```diff
@@ -0,0 +1,5 @@
+fennecversions = {
```

Comment:
This file isn't used by anything AFAICT. Is that a bug? If not, do we even need this?